test/units/modules/test_iptables.py

```diff
@@ -953,3 +953,38 @@ def test_destination_ports(self):
             '-m', 'comment',
             '--comment', 'this is a comment'
         ])
+
+    def test_match_set(self):
+        """ Test match_set together with match_set_flags """
+        set_module_args({
+            'chain': 'INPUT',
+            'protocol': 'tcp',
+            'match_set': 'admin_hosts',
+            'match_set_flags': 'src',
```

Comment:
Do we need additional test cases for `dst`, `dst,src`, and `src,dst`?

Reply:
Added an additional `src,dst` test. More test are probably overkill. Let me know if this is OK for you ...